src/status_im/chat/subs.cljs

```diff
@@ -53,6 +53,19 @@
   (fn [[chats id] [_ k chat-id]]
     (get-in chats [(or chat-id id) k])))
 
+(defn get-suggested-commands
```

Comment:
Looks like a good candidate for a quick test!

Reply:
Yeah, like mostly all subs in project :) Let's do this in separate PRs